server/.gitignore

```diff
@@ -57,3 +57,6 @@ uuid
 # Local development
 .opencode/
 .claude/
+
+# Gettext
```

Comment:
I would add a comment here we currently don't localize the app, so we gitignore it. And that we might revert that once we actually localize the dashboard.